### PR TITLE
Watt fixes

### PR DESCRIPTION
**Pull request by dhiltonp**
rawC should be positive. Previously the int32_t was being cast to a uint8_t. I'm surprised at how well it was working...

As a result, we had to change the mass divisor - rawC was ~2.5 times larger than it should have been, so we have to bump up the divisor by 2.5 times to have identical performance.

---------------

Also, if a very large set point was chosen, sometimes tempError overflowed - it's int16_t. Cap the values. Surprisingly, capping the error hasn't increased size at all! Good job, compiler.

### CODE REVIEW

File: workspace/TS100/src/main.cpp [hunk 1]
```diff
@@ -926,7 +926,7 @@ void startPIDTask(void const *argument __unused) {
 #ifdef MODEL_TS80
 	idealQCVoltage = calculateMaxVoltage(systemSettings.cutoutSetting);
 #endif
-	int32_t rawC = ctoTipMeasurement(100) - ctoTipMeasurement(101);  // 1*C change in raw.
+	uint8_t rawC = ctoTipMeasurement(101) - ctoTipMeasurement(100);  // 1*C change in raw.
```
Comment (Ralim) [inline workspace/TS100/src/main.cpp]: Why did you change this back to the uint8_t, would it not be better to let it stay as a int32_t, and then handle the overflow later on ?

Comment (dhiltonp) [reply workspace/TS100/src/main.cpp]: It's not a bounding issue, but due to casting a negative number into a uint8_t.

The overflow was from 100-101, which is -75 or so. That maps to 180 when cast to a uint8_t (I think?).

Anyway, it was messed up but within the bounds of a uint8_t.

Comment (dhiltonp) [reply workspace/TS100/src/main.cpp]: The fix actually shrunk the rom by about 20-30 bytes!

Comment (Ralim) [reply workspace/TS100/src/main.cpp]: Okay thats interesting, and GCC is running way above my head then 👍

**Pull request merged**